### PR TITLE
Completed Two of the checkpoint's outputs  

### PR DESCRIPTION
**Pull request by freemile2ru**
#  **What does this PR do?**

(1) This PR shows the proposed output to be submitted for the "setting up project repository checkpoint"
(2)This PR shows the proposed output to be submitted for the "setting up linters" checkpoint

### **Description of Tasks completed?**
    -Created the master branch and branched to the development branch
    -Shows how i described my commit messages following the andela style guide
    -Created an open pull request on personal Repo and decribed it following andela style guide
    -Edited ReadMe.md file
    -Ask fellows to comment

### **How should this be manually tested?**

N/A

### **Any background context I want to provide?**

the repository was reviewed by:

### **Name: Olawale aladeusi**
Comments:
    (1) credit for creating a development branch and keeping the master branch clean
    (2)commented on commit messages and shared the andela guide for writing commit messages
    (3) suggested set the development branch as the default branch
    (4) suggested i add description to my repository and the link to the hosted app 
    (5) lectured me on the appropriate times to commit my work

### **Name: jamiu abdulmujeeb**
Comments:
    (1) commended me on implementation of feedbacks gotten from Olawale
    (2) suggested i add security to my master branch
    (3) shared the andela guide for naming and describing a pull request

### **Description of Task completed?**
    -installed eslint on node
    -configured eslint to use es6 airbnb style
    -ran tests on my codes with eslint
    - screenshot the ouputs
    - linked my repository to hound cli site
    -created my hound.yml file and set my configuration to eslintrc
    -made a commit and pushed to my repository
    -screenshot hounded comment on my pull request

### **How should this be manually tested?**
     N/A
### **Any background context you want to provide?**
    N/A

### **Screenshots**
   [setting-up-linters](https://github.com/q-ode/test-simulations-mgmt-repo/tree/bayo-izanami-uibrahim/06-building-a-program-start-to-finish/setting-up-linters/output)

### CODE REVIEW

**Comment by freemile2ru** [comment]
Hello Bayo, 
please kindly help review this new PR as all corrections from the last PR reviews have been implemented. I have updated my read-Me file, created a new branch following the guideline format and have properly described the new PR to capture the two outputs being submitted. Thank you.

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+####Link to GitHub Repo
+- [ andela-uibrahim / invertedIndex](https://github.com/andela-uibrahim/inverted-index/pull/2)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md]: I believe I've addressed this issue of ReadMe files being empty.

Kindly address.

**Comment by freemile2ru** [comment]
Hello @q-ode I have updated this PR as required and have implemented all the feedbacks.
1) fixed my read me properly
2) include the proper links required
3) and I have resolved all merged conflicts

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+
+
+##HoundCL comment on PR
+
+- [andela-uibrahim/ HoundCL](https://photos.google.com/photo/AF1QipO2HMwbEauOVygu7Ebitu8VCAwObsaljeqv-d4)
+<img src="https://lh3.googleusercontent.com/IWNUEcibgcKo1vA-2XiSNmDbD1UM5cHxuxuWk-RK-RphPkibbM7KQelCFZh9MHQh1W0OfeKSI57vblGPE3dZiaHUX5rY5JhGFNUZ7EpxNz71g5DkHqeOu-US9Axkie2jZ8uK0hhic3lunkxBlJVvx_Qga80MZ83Ru3yg15LF9aQSTDWMvF2ay3Y-stiN_R0rkb7hUuY9Pc4s0t3XT49z8Wnern4XhmOK9xZMJqAOI_sAMVCYgDC85fWuB-7UgH-Iy8HCXuHj8O87l2Bgbudnleu5ZWF8ZN3F_A26j0qmD_KxMJOtmU0U0UDiurvrwlVO_eFSgCOyUYjVLuNAKtysKTQEp_RkzRVPNyeRUH_8Z9dOXkv1E8g9x5o_0R5SR1sHXcyg2KYgFp99ioAZJ9aTJ7moiK6zyAa_kIrr10EW1tpgLqZtmAAaeNaFK3ordZesrf7tuo8bUP6iaX950987M6D_0g3TH5m5bTySxekJCSbRsslG0v0BYfnyX19WXG5__xM--3NvS8oywS1oFK01_rR4nmgSawglW-HKm_p9J9G1W9BQ6i56RHfliFVyj2yNuI3sEtUKdSVbZvzQfqcry4wN2SS2Uv8Kb_0izdA27OWF=w964-h659-no" />
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md]: What are you trying to achieve here?

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+
+####Link to pull request from branch setting-up-repository to development branch
+- [ andela-uibrahim / invertedIndex](https://github.com/andela-uibrahim/inverted-index/pull/2)
+
+####Link to GitHub Repo
+- [ andela-uibrahim / invertedIndex](https://github.com/andela-uibrahim/inverted-index/tree/chore/1/setting-up-repository)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:11]: There's something wrong with your `ReadMe`.

The Markdown format is showing without the appropriate representation.

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md (hunk 2 above)
Comment (freemile) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md]: @q-ode  I was trying to upload the images to view on my read me file. but I have rectified that now

File: 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md (hunk 3 above)
Comment (freemile) [inline 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md]: @q-ode changes have been implemented please kindly review.

Comment (freemile2ru) [reply 06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md:11]: @q-ode changes have been implemented please kindly review

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md (hunk 2 above)
Comment (freemile) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md]: @q-ode I was trying to upload the images to view on my read me file. but I have rectified that now

Comment (freemile2ru) [reply 06-building-a-program-start-to-finish/setting-up-linters/output/README.md]: @q-ode I was trying to upload the images to view on my read me file. but I have rectified that now

**Comment by q-ode** [comment]
Nice one!

**Pull request merged**